### PR TITLE
Update visualize-force-graph, add more arguments

### PR DESCRIPTION
**Pull request by GrantXie**

### CODE REVIEW

File: kgtk/cli/visualize-force-graph.py [hunk 1]
```diff
@@ -117,266 +275,432 @@ def run(input_file: KGTKFiles,
...
-                    s = 'rgba' + str(tuple(int(h[i:i+2], 16) for i in (0, 2, 4)))
-                    d['nodes'].append({'id': df['node;label'][i][1:df['node;label'][i].find('@')-1], 'color': s, 'fx': float(df['x'][i]), 'fy': float(df['y'][i])})
-                else:
-                    h = df['color'][i].lstrip('#')
-                    s = 'rgba' + str(tuple(int(h[i:i+2], 16) for i in (0, 2, 4)))
-                    d['nodes'].append({'id': df['node;label'][i], 'color': s})
+                d['nodes'].append({'id': ele[0], 'label': ele[1]})
         else:
-            df = pd.read_csv(node_file)
+            if '.tsv' in node_file:
```
Comment (saggu) [inline kgtk/cli/visualize-force-graph.py:443]: Use `KGTKReader` for reading node or edge files, it can take care of this file extension logic

**Review by saggu — CHANGES_REQUESTED** [review]
Change all `print` statements, write them out to `std.err` instead

File: kgtk/cli/visualize-force-graph.py [hunk 2]
```diff
@@ -117,266 +275,432 @@ def run(input_file: KGTKFiles,
...
+        if node_size_minimum == 0.0:
+            node_size_minimum = 0.01
+
+        if edge_width_minimum == 0:
+            edge_width_minimum = 0.01
+
+        if 'node1' not in kr.column_name_map:
+          print('No node1 col in edge file')
+        if 'node2' not in kr.column_name_map:
+          print('No node2 col in edge file')
```
[the remark this answers is not in the document]
Comment (GrantXie) [reply kgtk/cli/visualize-force-graph.py]: have all messages print to stderr now

File: kgtk/cli/visualize-force-graph.py (hunk 1 above)
Comment (GrantXie) [reply kgtk/cli/visualize-force-graph.py]: changed to KGTKReader

File: kgtk/cli/visualize-force-graph.py [hunk 3]
```diff
@@ -117,266 +275,432 @@ def run(input_file: KGTKFiles,
         d = {}
         nodes = set()
         edges = []
+        base = math.e
+
+        if node_size_minimum == 0.0:
```
[the remark this answers is not in the document]
Comment (GrantXie) [reply kgtk/cli/visualize-force-graph.py:283]: this is to make sure the user won't input a  node_size_minimum of 0

[the remark this answers is not in the document]
Comment (saggu) [reply kgtk/cli/visualize-force-graph.py:283]: instead of changing the user input, you should raise an error @GrantXie

**Pull request closed without merging**